.github/workflows/draft-release.yml

```diff
@@ -0,0 +1,199 @@
+name: Cli Draft Release
+on:
+ workflow_dispatch:
+    inputs:
+      change-log:
+        description: 'Open API Changelog.'
+      version-type:
+        description: 'Version to upgrade, Major: 0, Minor:1, Patch: 2'
+      homebrew-branch:
+        description: 'HomeBrew branch name'
+        default: main
+      homebrew-prerelease:
+        description: 'HomeBrew prerelease'
+        default: 'true'
+      draft-release:
+        description: 'Draft-release flag (true/false)'
+        default: 'true'
+
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [14.x, 10.x]
+    steps:
+      - name: Checkout cli repo
+        uses: actions/checkout@v2
+      - run: make install
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v2
+        with:
+          node-version: ${{ matrix.node-version }}
+          cache: 'npm'
+      - name: Run tests
+        run: npm test
+  update-api-definitions-changelog:
+    runs-on: ubuntu-latest
+#     needs: [test]
+    steps:
+      - name: Checkout cli
+        uses: actions/checkout@v2
+        with:
+          token: ${{ secrets.REPO_ACCESS_TOKEN }}
+      - name: Update Changelog
+        run: |
+          bash .github/scripts/commit-api-spec-change-log.sh "${{ github.event.inputs.change-log }}" ${{ github.event.inputs.version-type }}
+  release:
+    runs-on: ubuntu-latest
+    needs: [update-api-definitions-changelog]
+    outputs:
+      draft-tag-name: ${{ steps.semantic-release-draft.outputs.TAG_NAME }}
```

Comment:
Are we planning the clean the draft related code from main?

Reply:
Yes. [DII-309](https://issues.corp.twilio.com/browse/DII-309). This would also involve renaming some files in this branch so that merge with main is smooth without conflicts.